INSTALL.md

```diff
@@ -78,21 +81,35 @@ $ camlp5 -v
 Camlp5 version 7.03 (ocaml 4.02.3)
 ```
 
-Under Mac OS X, you may obtain ProofGeneral from http://proofgeneral.inf.ed.ac.uk/.
-It comes with installation instructions.  Your version of emacs determines
-which version of ProofGeneral you need, roughly, so some experimentation may be
-required; you may even need the current development version if your emacs is
-recent.
-
-
 ## Preparing for the installation under Linux
 
-Under Ubuntu or Debian, you may install ocaml (and ProofGeneral) with
+uUnder Ubuntu or Debian, you may install ocaml (and ProofGeneral) with
 
 ```bash
 $ sudo apt-get install build-essential git ocaml ocaml-nox ocaml-native-compilers camlp4-extra camlp5 proofgeneral proofgeneral-doc libgtk2.0 libgtksourceview2.0 liblablgtk-extras-ocaml-dev ocaml-findlib
 ```
 
+Arch Linux and Manjaro Linux:
+
+    To prepare for compiling it yourself:
+
+      sudo pacman --sync --needed archlinux-keyring
+      sudo pacman-key --populate archlinux
+      sudo pacman --sync --needed ocaml camlp5 ocaml-findlib
+
+    Install ProofGeneral from its website as described above.  Or,
+    alternatively, install it from AUR (
+    https://aur.archlinux.org/packages/proofgeneral/ ) using an auxiliary
+    package manager such as "cower" ( https://aur.archlinux.org/packages/cower/
+    ) or yaourt ( https://aur.archlinux.org/packages/yaourt/ ).  That approach
+    will be especially convenient for expert Arch Linux users who find other
+    packages at AUR worth installing.
+
+    To upgrade the system:
+
+      sudo pacman-key --refresh-keys
+      sudo pacman --sync --refresh --sysupgrade
```

Comment:
I usually just type `sudo pacman -Syu` or even better `yaourt -Syu`

Reply:
Non-expert users like me would find such abbreviations confusing.  Expert users know the abbreviations and can use them.

Reply:
Maybe... But if you read the `man` page for pacman the example for how to update the package list and upgrade all packages is simply `pacman -Syu`, so I think that is the standard way to do things. I've been using Arch for about 10 years and I very rarely (like 2 times per year) have to run `pacman-key`.

Reply:
I had to run `pacman-key` today, for the first time, hence the instruction about that.  If it saves someone (such as me in the future) some trouble once, then it's worth it.

Reply:
Strange. Have you installed archlinux-keyring (https://www.archlinux.org/packages/core/any/archlinux-keyring/) ?

Reply:
Yes.  Indeed, that's also in the instructions.

Reply:
... it kept downloading package files and claiming that the PGP key was untrusted, so it couldn't use the files.

Reply:
Ok, very strange. That never happened to me...